src/schemas/tasks.schema.ts

```diff
@@ -0,0 +1,11 @@
+import { z } from 'zod';
+
+export const taskSchema = z.object({
+  id: z.string(),
+  title: z.string(),
+  description: z.string(),
```

Comment:
i think description could be optional
```suggestion
  description: z.string().optional(),
```